Jenkinsfile

```diff
@@ -104,20 +107,20 @@ node ('docker') {
             } catch (err) {
                 currentBuild.result = "FAILURE"
             } finally {
-                sendhipchat()
+                sendhipchat(branchName)
             }
         }
     } finally {
         sh "./stop-selenium.sh"
     }
 }
 
-def sendhipchat() {
+def sendhipchat(branchName) {
     res = currentBuild.result
     if(res == null) {
         res = "SUCCESS"
     }
-    message = "ATH: ${env.JOB_NAME} #${env.BUILD_NUMBER}, status: ${res} (<a href='${currentBuild.absoluteUrl}'>Open</a>)"
+    message = "ATH: ${env.JOB_NAME} #${env.BUILD_NUMBER} branch: ${branchName}, status: ${res} (<a href='${currentBuild.absoluteUrl}'>Open</a>)"
```

Comment:
Maybe "run against: blueocean/${branchName}" would be a bit clearer in tersm of what that means to someone reading it.